pyi.Dockerfile

```diff
@@ -100,6 +100,14 @@ RUN ./bcc_helpers_build.sh
 # bcc & gprofiler
 FROM centos${GPROFILER_BUILDER} AS build-stage
 
+# fix repo links for CentOS 8, and enable powertools (required to download glibc-static)
+RUN if grep -q "CentOS Linux 8" /etc/os-release ; then \
```

Comment:
Docs in lines 17-19 in this file is somewhat missleading.

Besides, if we have differences in the container versions we pass to `pyi.Dockerfile` between x86_64 and aarch64, then maybe not have a default for them in `pyi.Dockerfile` and have the comment and sha256 of the container in `build_x86_64_container.sh`?

Reply:
Fair

Reply:
Fixed & tested build before / after, seems okay (result is cached so I didn't fuck up any base layer etc)